### ISSUE TITLE
Package install error

### ISSUE THREAD

**Issue by bguthrie** [fauna-labs/fauna-schema-migrate#90]
Hi folks—I'm getting the following error when I attempt to install the package:

```
❯ yarn add @fauna-labs/fauna-schema-migrate
➤ YN0000: ┌ Resolution step
➤ YN0001: │ Error: ink-select-input@github:fauna-labs/ink-select-input: Unknown head ("master")
```

I suspect it may have been caused by this upstream pull request: https://github.com/fauna-labs/ink-select-input/pull/1

Thanks in advance for any assistance you can provide here—am I missing something?

**Comment by rts-rob** [comment]
Thanks for reporting, @bguthrie. You're right - we are in the process of moving all default branches to `main` - looks like this got caught up in that.

Will resolve shortly.

**Comment by rts-rob** [comment]
@bguthrie looking at this locally, it seems this may be coming from the yarn lockfile, not the config itself. I am unable to reproduce with a new project running yarn v1.22.10 - it resolves as follows:

```
"ink-select-input@github:fauna-labs/ink-select-input":
  version "4.2.1"
  resolved "https://codeload.github.com/fauna-labs/ink-select-input/tar.gz/7eb9142a5401bbc492842d0fc26aefa5f07b8f5d"
  dependencies:
    arr-rotate "^1.0.0"
    figures "^3.2.0"
    lodash.isequal "^4.5.0"
```

What version of yarn/npm/os are you running?

**Comment by bguthrie** [comment]
Thanks for the quick turnaround, @rts-rob. I'm running yarn 2.4.2 with a couple of discrete workspaces. My local lockfile doesn't contain a reference to `fauna-schema-migrate` or `ink-select-input`, which I suppose stands to reason. Will investigate more shortly.

**Comment by rts-rob** [comment]
Hi @bguthrie - did you get a chance to come back to this? Is it still throwing an error?

**Comment by Shadid12** [comment]
#99

**Comment by rts-rob** [comment]
Fixed in #99 (merged to main)
